script/tracker_node.py

```diff
@@ -40,6 +40,12 @@ def image_callback(self, msg):
         header = msg.header
         encoding = msg.encoding
         numpy_image = ros_numpy.numpify(msg)
+
+        # Convert RGBA to RGB
+        if numpy_image.shape[2] == 4:  # Check if the image has 4 channels
+        	numpy_image = cv2.cvtColor(numpy_image, cv2.COLOR_RGBA2RGB)
+        	encoding = "bgr8"
```

Comment:
- Your code uses tab for indentation, but please unify it with 4 spaces.
- `cv2.COLOR_RGBA2RGB` converts from RGBA to RGB and outputs in RGB order. However, `encoding = "bgr8"` indicates BGR order. 
```suggestion
            numpy_image = cv2.cvtColor(numpy_image, cv2.COLOR_RGBA2RGB)
            encoding = "rgb8"
```